### PR TITLE
Fetch missions

### PR DESCRIPTION
**Pull request by Bushido-brown**
Fetch Missions

Fetched data from the Missions endpoint (https://api.spacexdata.com/v3/missions) when a user navigates to the Missions section.   

dispatch an action to store the selected data in Redux store:

- mission_id
- mission_name
- description

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import store from './redux/configureStore';
+
+// const root = ReactDOM.createRoot(document.getElementById('root'));
+// root.render(
+//   <React.StrictMode>
+//     <Router>
+//       <App />
+//     </Router>
+//   </React.StrictMode>,
+// );
```
Comment (jevazquezb) [inline src/index.js:17]: Kindly, remove this commented code.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```
Comment (jevazquezb) [inline src/index.js:7]: Please, remove this line since you're not using this file.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <Provider store={store}>
+        <App />
+      </Provider>
+    </Router>
+  </React.StrictMode>,
+);
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (jevazquezb) [inline src/index.js:31]: Kindly, remove this comment and the function from the end since you are not using it.

**Review by jevazquezb — CHANGES_REQUESTED** [review]
# STATUS: Changes Required ♻

Hi @Bushido-brown,

**Your contribution is looking great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.**
___ 
### Great job!!
- Your git-flow is correct. ✔️
- JS, HTML and CSS best practices. ✔️
- You used store, actions and reducers.
- You implemented very well the async functions. ✔️
- You handled events properly. ✔️

___ 
### ♻ Request Changes ♻

- [ ] _Check the comments under the review._
___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _Check the comments under the review._
___

### Cheers and Happy coding! 💻 👏
___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/index.js (hunk 3 above)
Comment (Bushido-brown) [reply src/index.js:31]: Thanks for the corrections, I'll go ahead and implement them right away.

Comment (Bushido-brown) [reply src/index.js]: I have made the required changes

**Review by jevazquezb — APPROVED** [review]
# STATUS: Approved 🟢

Hi @Bushido-brown,

Your project is complete! There is nothing else to say other than... it's time to merge it
Congratulations! 🎉
___ 
### Great job!!
- Your git-flow is correct. ✔️
- JS, HTML and CSS best practices. ✔️
- You used store, actions and reducers.
- You implemented very well the async functions. ✔️
- You handled events properly. ✔️
___

### Cheers and Happy coding! 💻 👏
___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

**Pull request merged**